Activities/Cryptography/UiPath.Cryptography.Activities/EncryptFile.cs

```diff
@@ -64,6 +64,11 @@ public EncryptFile()
         [LocalizedDescription(nameof(Resources.Activity_EncryptFile_Property_OutputFilePath_Description))]
         public InArgument<string> OutputFilePath { get; set; }
 
+        [LocalizedCategory(nameof(Resources.Input))]
+        [LocalizedDisplayName(nameof(Resources.Activity_EncryptFile_Property_OutputFileName_Name))]
+        [LocalizedDescription(nameof(Resources.Activity_EncryptFile_Property_OutputFileName_Description))]
+        public InArgument<string> OutputFileName { get; set; }
```

Comment:
please add similar to Decrypt File as well

Reply:
Added